components/popover/blender/script.js

```diff
@@ -0,0 +1,22 @@
+function GELtoggelPopover($panel) {
+	var className = $panel.attr('class');
+	var classBits = className.split('-open');
+
+	if (classBits.length > 1) {
+		$panel.attr('class', classBits.join(''));
+	} else {
+		var items = className.split('-');
+		items.splice(items.length - 1, 0, 'open');
+		$panel.attr('class', items.join('-'));
+	}
+}
+
+$(function () {
+	$('[data-js="popover__version__"]').on('click', function () {
+		GELtoggelPopover($(this).next('[data-js="popover-panel__version__"]'));
+	});
+
+	$('[data-js="popover-closeBtn__version__"]').on('click', function () {
+		GELtoggelPopover($(this).parents('[data-js="popover-panel__version__"]'));
```

Comment:
`$.parents()` ? 

I think `$.closest()` is the recommended way to get the matching ancestor

Reply:
My jQuery is rusty at best... I figured `parents` would only look down the tree while `closest` looks both ways?

Reply:
`closest()` selects the first element that matches the selector, up from the DOM tree. Begins from the current element and travels up.

`parent()` selects one element up (single level up) the DOM tree.

`parents()` method is similar to parent() but selects all the matching elements up the DOM tree. Begins from the parent element and travels up.

So one could argue that `parents()` is more performant because it starts at the parent therefor has one less iterations 😬

Reply:
Up to you. I was under the impression that if there were additional matching elements up the tree it would select all, rather than the one as we'd expect.